### DISCUSSION TITLE
Releasing from both `master` and `maint` branches

### DISCUSSION THREAD

**Discussion by jwodder** [intuit/auto#1981] [Q&A]
I'm in the process of setting up auto for a project that makes releases from two different branches: new major & minor versions are released from `master`, while new patch versions are released from `maint`.  How do I properly configure `auto` to handle this?  I have `baseBranch` set to `master`.  Should `versionBranches` be set to something?  The documentation on that configuration option is incredibly lacking.

**Comment by hipstersmoothie** [comment]
You don't need to set it to master, that's the default so you should be fine there. As for releasing patches from another branch I'm not sure that's a use case auto really supports. How do merges between the branches work? `versionBranches` is more for when you cut a major and still want to patch old major versions
